chain/ethereum/src/env.rs

```diff
@@ -182,7 +182,7 @@ struct Inner {
         default = "100"
     )]
     target_triggers_per_block_range: u64,
-    #[envconfig(from = "GRAPH_ETHEREUM_GENESIS_BLOCK_NUMBER", default = "0")]
+    #[envconfig(from = "GRAPH_ETH_CALL_NO_GAS", default = "421613,421614")]
```

Comment:
I think you meant to change the line below

Reply:
yeah right.